### DISCUSSION TITLE
When an item added to cart show a success message with product name and quantity added

### DISCUSSION THREAD

**Discussion by icreatestuff** [craftcms/commerce#3667] [Q&A]
When a customer adds a product to their cart I need to be able to show a modal or similar that indicates the success including details about the specific product and quantity just added.

The `commerce/cart/update-cart` action provides a successful response but doesn't provide any detail about what was added as far as I can tell.

A workaround that gets about 90% of the way is to pass the product ID from the current page to the modal and then loop over all the cart line items only outputting the one with a matching product ID. _However_, the quantity shown is the current total quantity of that item in the cart, rather than the quantity **_just_** added. I may have added 1 already and then subsequently added another. The modal should indicate that 1 was just added, but with my current implementation will show 2 just added.

![Screenshot 2024-09-11 at 16 38 17](https://github.com/user-attachments/assets/7dd4125d-217a-4c7c-abaa-7287aa47c77f)

Are there any existing ways to get something like this? Thanks

**Comment by lukeholder** [comment]
@icreatestuff sorry for delay on response for this one.

If you are doing an ajax request to the `commerce/cart/update-cart` action, you should get a `cart` object in the response. Can you post the http response so we take a look? Thanks!
